model/patch/github.go

```diff
@@ -32,66 +33,97 @@ type Intent interface {
 	GetType() string
 }
 
-// githubIntent represents an intent to create a patch build as a result of a
+// GithubIntent represents an intent to create a patch build as a result of a
 // PullRequestEvent webhook. These intents are processed asynchronously by an
 // amboy queue.
-type githubIntent struct {
+type GithubIntent struct {
```

Comment:
I don't see the case where you needed to make this public.

Reply:
See the MockPatchIntentConnector; I needed to use the msg_id to guarantee that duplicate messages wouldn't result in two patch intents. From talking to @bsamek, he didn't want to add a function to the interface for exposing id

Reply:
I think its fine to have a method exposing ID and leaving the model private. If we're serious about using an interface for patch intents, then I think we should stick to that, but lets get that squared away with him before his vacation.